sdk/RELEASE.md

```diff
@@ -11,6 +11,21 @@
 
 ## Bug Fixes and Other Changes
 
+## Documentation Updates
+# 1.8.16
+## Major Features and Improvements
+
+## Breaking Changes
+
+### For Pipeline Authors
+
+### For Component Authors
+
+## Deprecations
+
+## Bug Fixes and Other Changes
+* Relax required google-auth version [#8470](https://github.com/kubeflow/pipelines/pull/8470)
```

Comment:
nit: can be more specific, e.g.: `Require google-api-core>=1.31.5, >=2.3.2 [#7377](https://github.com/kubeflow/pipelines/pull/7377)`

Reply:
Good thinking. Updated.